src/mtcp/mtcp_restart.c

```diff
@@ -104,11 +104,13 @@ typedef struct RestoreInfo {
   VA vvarStart;
   VA vvarEnd;
   fnptr_t post_restart;
+  fnptr_t post_restart_debug;
```

Comment:
Please update the offset of `restorememoryareas_fptr` in the `restart_fast_path()` function for the Intel compiler (`#if defined(__INTEL_COMPILER) ...`).